README_WCTE.txt

```diff
@@ -0,0 +1,13 @@
+###############################################################
```

Comment:
I guess we don't want to pollute WCSim with too many detector-specific files, but since HK is moving on from WCSim, I guess this could just be moved into `docs`, or copied into the `.mac` instead which already has most of the info here.

Reply:
I am currently changing these things within my own repo. I have moved this into /doc